source/server/getting-started/migrating.rst

```diff
@@ -153,8 +193,8 @@ Migrating to SpongeForge
 ------------------------
 
 You must first ensure you are running a version of Forge that is compatible with the version of Sponge your plan to use.
```

Comment:
It may be old, but it's still a typo: "your" -> you. Also, Sponge -> SpongeForge.